source/documentation/guidance/buildpacks.md

```diff
@@ -1,44 +1,120 @@
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					Physical hardware
+					<br>- resolve hardware failures
+					<br>- encrypt network data traffic
+				</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+
+### Standard buildpacks
+
+You are only responsible for the app code and its dependencies. This includes [penetration testing](https://docs.cloud.service.gov.uk/#penetration-testing) your app.
 
-#### Standard buildpack
-You are only responsible for the app code and its dependencies.
+If you're relatively new to cloud or have a small development team, we recommend using the standard buildpacks to maximise the support you will receive from GOV.UK PaaS.
 
-#### Custom buildpacks
+### Custom buildpacks
 
-You are responsible for managing this custom buildpack as well as the app code and its dependencies. This applies both in setting up the app, and running it in production.
+You are responsible for managing this custom buildpack's language and runtime, as well as the app code and its dependencies. This applies both in setting up the app, and running it in production.
 
 When setting up the app:
 
 - Do your own due diligence on finding a maintained and supported custom buildpack so that you have a community to call upon in the case of problems
-- Test deploying with the buildpack early in your build process, not just before go-live
+- Test deploy with the buildpack early in your build process, not just before go-live
 
 When running the app in production:
 
 - Ensure you have sufficient time to fix issues yourself as using them for deadline-driven development may be risky
 - Regularly maintain your buildpack to update your runtime and dependencies as new security vulnerabilities are discovered and fixed
-- We may not be able to offer support for P1 or out-of-hours incidents; standard SLAs will not apply
+- We may not be able to offer support for P1 or out-of-hours incidents and standard SLAs will not apply
+
+Due to these increased responsibilities, we recommend that, if you must use a custom buildpack, you use a community supported one that is maintained and updated by that community. We do not recommend that you create your own buildpack.
 
-#### Docker images
+Contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) if you are interested in this feature. If a custom buildpack is requested by multiple tenants, we will look to improve our support in this area.
+
+### Docker images
```

Comment:
It seems strange that we detailed standard and custom buildpacks here, but Docker is only a short sentence and a link. Do you think we should move some of the stuff from the Docker section to here?

Reply:
I agree but I think that should be looked at as part of an overall change of the Docker section as it is not part of this story. I think the Docker section overall needs some work, so I recommend we draw a line under this and look at this later. What do you think?

Reply:
I'm fine with that.

Reply:
👍